unit-test

```diff
@@ -5,7 +5,7 @@ cd $here
 
 if [ $# -eq 0 ] ; then
 	echo "Launching unit tests in docker"
-	TEST=1 exec ./dev-shell ./unit-test indocker
+	TEST=1 exec ./dev-shell sudo -u testuser ./unit-test indocker
```

Comment:
weird, but it works, so ship it.

Reply:
Yes, but I managed to consistently reproduce it locally. Something must have changed recently, maybe in docker for ubuntu.

I need at least one approval to merge it.